packages/db/src/main/cfxjs/db/queries.cljs

```diff
@@ -1762,22 +1762,44 @@
      :currentNetwork cur-net
      :accountGroups  data}))
 
-(defn- sort-tx [[noncea txa _] [nonceb txb _]]
-
+(defn- sort-tx [[noncea _] [nonceb _]]
```

Comment:
I have a question, I didn't find the code “sorted by 'Completed > Executed > Pending > Failed'”

Reply:
It's at https://github.com/Conflux-Chain/helios/blob/1d0f247c78401b3246b3c0a809b4b8e557b41ebf/packages/db/src/main/cfxjs/db/queries.cljs#L1782

It's sort by the value of status, which is a number
 https://github.com/Conflux-Chain/helios/blob/1d0f247c78401b3246b3c0a809b4b8e557b41ebf/packages/background/src/db-schema.js#L152